rod_cutting_problem/rod_cutting.c

```diff
@@ -0,0 +1,32 @@
+/*
+*  Given a rod of length n length unit and an array of prices that contains prices of all pieces of size smaller than n. 
```

Comment:
@Monal5031 Please send a PR with the fix.. 
Seriously, should have read the comments too.